lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -460,7 +460,10 @@ instance
                 <> show (natVal (Proxy :: Proxy mw))
                 <> " words are expected."
             ErrDictionary (ErrInvalidDictionaryWord w) ->
-                "Found invalid (non-English) word: \"" <> B.toList w <> "\"."
+                "Found an unknown word not present in the pre-defined dictionary\
+                \: \"" <> B.toList w <> "\". The full dictionary is available \
+                \here: https://github.com/input-output-hk/cardano-wallet/tree/m\
+                \aster/specifications/mnemonic/english.txt"
```

Comment:
**[remark]** 

Alternatively, I just thought that the dictionary is also available here: https://github.com/input-output-hk/cardano-crypto/blob/master/src/Crypto/Encoding/BIP39/English.hs#L33

Reply:
ok, thanks. I will stick to the current solution. This one w control. The one in cardano-crypto can change (for example it can no longer be line number 33)